build/drivers/dfu_trigger_and_cdc_acm/nrfconnect_cdc_acm.inf

```diff
@@ -39,6 +39,7 @@ CatalogFile = nrfconnect_cdc_acm.cat
 
 ;----------------------------------------------------------
 ; Windows 2000, XP, Vista, Windows 7, Windows 8 - 32bit and 64bit
+; Based on https://docs.microsoft.com/en-us/windows-hardware/drivers/usbcon/usb-driver-installation-based-on-compatible-ids
```

Comment:
This link is already at the top of the document. Intentional to have it twice?

Reply:
No, I completely missed that :-|

Maybe I should rewrite the comments in both PRs and put everything at the top of the file